PhysicsTools/PatAlgos/plugins/PATLeptonTimeLifeInfoProducer.cc

```diff
@@ -169,6 +170,10 @@ void PATLeptonTimeLifeInfoProducer<T>::produceAndFillIPInfo(const T& lepton,
     AnalyticalImpactPointExtrapolator extrapolator(transTrack.field());
     TrajectoryStateOnSurface closestState =
         extrapolator.extrapolate(transTrack.impactPointState(), RecoVertex::convertPos(pv.position()));
+    if (!closestState.isValid()) {
+      edm::LogError("PATLeptonTimeLifeInfoProducer") << "closestState not valid!";
```

Comment:
please also print `transTrack.impactPointState()` and  `RecoVertex::convertPos(pv.position())`